xtask/src/release.rs

```diff
@@ -78,8 +158,18 @@ impl flags::Release {
         Ok(changes.trim().to_owned())
     }
 
+    /// Get the name of this release.
+    fn get_name(&self) -> String {
+        format!("{} {}", self.name, self.version)
+    }
+
+    /// Get the path of the crate for this release.
+    fn get_path(&self) -> PathBuf {
+        self.project_root.join(&self.name)
+    }
+
     /// Load the GitHub config from the config file.
-    fn get_remote(&self) -> Result<String> {
+    fn get_remote() -> Result<String> {
```

Comment:
```suggestion
    fn remote() -> Result<String> {
```

or maybe `git_remote`.